broker/src/main/java/io/camunda/zeebe/broker/transport/backpressure/PartitionAwareRequestLimiter.java

```diff
@@ -113,12 +112,12 @@ private static GradientLimit getGradientLimit(final GradientCfg gradientCfg) {
         .build();
   }
 
-  private static AIMDLimit getAIMD(final AIMDCfg aimdCfg) {
-    return AIMDLimit.newBuilder()
+  private static StabilizingAIMDLimit getAIMD(final AIMDCfg aimdCfg) {
+    return StabilizingAIMDLimit.newBuilder()
         .initialLimit(aimdCfg.getInitialLimit())
         .minLimit(aimdCfg.getMinLimit())
         .maxLimit(aimdCfg.getMaxLimit())
-        .timeout(aimdCfg.getRequestTimeout().toMillis(), TimeUnit.MILLISECONDS)
+        .expectedRTT(aimdCfg.getRequestTimeout().toMillis(), TimeUnit.MILLISECONDS)
```

Comment:
:question: Is it correct that we use here the request timeout as RTT?

Reply:
Yes. timeout is that max RTT allowed at which aimd detects contention and starts reducing the limit.